tools/wave/tox.ini

```diff
@@ -1,14 +1,13 @@
 [tox]
-envlist = py38,py39,py310,py311,py312
+envlist = py36,py37,py38,py39
 skipsdist=True
 skip_missing_interpreters = False
 
 [testenv]
 deps =
   -r{toxinidir}/../requirements_pytest.txt
-  -r{toxinidir}/requirements.txt
   -r{toxinidir}/../wptrunner/requirements.txt
-  -r{toxinidir}/../wptrunner/requirements_chromium.txt
+  -r{toxinidir}/../wptrunner/requirements_chrome.txt
```

Comment:
I believe this should be reverted, the filename is [requirements_chromium.txt](https://github.com/web-platform-tests/wpt/blob/master/tools/wptrunner/requirements_chromium.txt)